tools/build/workspace.bzl

```diff
@@ -211,7 +211,7 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
             native.android_ndk_repository,
             name = "androidndk",
             locals = locals,
-            api_level = 21, # This is the minimum API
+            api_level = 23, # This is the minimum API
```

Comment:
Did we run into issues with the _armeabi memclr? .... stuff when increasing this? I thought there was some problem on older devices.

Reply:
I think the memclr was from a later version, but effectively, this would mean that, at least for on-device replay, the minimum required version would be M, whereas currently it may work on L.